editor/components/inserter/index.js

```diff
@@ -79,19 +80,25 @@ class Inserter extends Component {
 						onClose();
 					};
 
-					return <InserterMenu onSelect={ onSelect } />;
+					return <InserterMenu onSelect={ onSelect } supportedBlockTypes={ supportedBlockTypes } />;
```

Comment:
It looks like this prop is not used in the current component, can we move to the menu component instead?

Also might be good to factorize, using the selector proposed above?

Reply:
Hi @youknowriad, the prop supported block types is used to compute hasSupportedBlocks flag that's why it was computed in this component.
Regarding the factorize to selector, I will try to make this more generic maybe it will involve selector plus HoC that accesses the context.